### DISCUSSION TITLE
Interactive Services on Windows 10 Enterprise 21H2?

### DISCUSSION THREAD

**Discussion by simpsonetti** [Ylianst/MeshCentral#5549] [Issues]
Hi there, i still try to test Meshcentral. So some funtions and the usability is clearer to me now. 
But now i figuring my main problem here at our business: 
The Agent will be blocked. I'm wondering that i can connect to the maschine and use rdp, but not with the agent. So i found here in the event viewer:
The Service "Mesh Agent" is marked as interactive Service. However the System has been configured so that interactive services are not possible. The Service may not function properly. 

So i read something about that and there are many sites to "enable" it. But whats the way that it won't be shut down in a next update? 
Where to enable it the rigth way?
Regards Sebastian

**Comment by dinger1986** [comment]
Why don't you allow exceptions?

**Reply by simpsonetti** [reply]
Well, i'not in our windows-team. I think there are some restritcions to avoid malware.
So i have to ask our windows-team to seperate the installation to my needs to get  mesh agent running. But therefor i want to know what this causes and not open a bigger hole.

**Reply by dinger1986** [reply]
Most likely the exe is unsigned avs wont like it

**Reply by simpsonetti** [reply]
And this causes the message in the event viewer?

**Reply by dinger1986** [reply]
Could be, or some other group policy, or other policy
